### PR TITLE
update benchmarks/ locust versions + small fixes

### PR DESCRIPTION
**Pull request by annapendleton**
Change includes:
- Small README.md fixes
- locust requirements.txt updated
- any small changes needed to make end to end work

Validated with end to end deployment of default terraform.tfvars with infra/, text-generation-inference/, and locust-load-inference/

### CODE REVIEW

**Comment by annapendleton** [comment]
@imreddy13 @achandrasekar @andrewsykim PTAL

File: benchmarks/benchmark/tools/locust-load-inference/README.md [hunk 1]
```diff
@@ -8,10 +8,12 @@
     - [Step 2: create and give service account access to view dataset](#step-2-create-and-give-service-account-access-to-view-dataset)
     - [Step 3: create output bucket](#step-3-create-output-bucket)
     - [Step 4: create and give service account access to write to output gcs bucket](#step-4-create-and-give-service-account-access-to-write-to-output-gcs-bucket)
+    - [Step 6: create and configure terraform.tfvars](#step-6-create-and-configure-terraformtfvars)
```
Comment (achandrasekar) [inline benchmarks/benchmark/tools/locust-load-inference/README.md:11]: Can you change the numbering? It stays step 6.

File: benchmarks/benchmark/tools/locust-load-inference/README.md [hunk 2]
```diff
@@ -51,13 +53,13 @@ You will set the `gcs_path` in your `terraform.tfvars` to this gcs path containi
 
 ### Step 2: create and give service account access to view dataset
 
-The Locust workload requires storage.admin access to view the dataset in the given gcs bucket. If you are running with workload identity, it obtains this access via a kubernetes service account that is backed by a gcloud service account. If you followed steps in `../../infra`, then you already have a kubernetes and gcloud service account created that you can use here.
+The Locust workload requires storage.admin access to view the dataset in the given gcs bucket. If you are running with workload identity, it obtains this access via a kubernetes service account that is backed by a gcloud service account. If you followed steps in `/infra`, then you already have a kubernetes and gcloud service account created that you can use here.
```
Comment (achandrasekar) [inline benchmarks/benchmark/tools/locust-load-inference/README.md:56]: `../../infra` is the correct location right? Why change it?

Comment (annapendleton) [reply benchmarks/benchmark/tools/locust-load-inference/README.md:56]: we utilized "infra/" across the other locations so this change is to make it consistent - I can change those to use the relative path as desired.

File: benchmarks/benchmark/tools/locust-load-inference/README.md (hunk 1 above)
Comment (annapendleton) [reply benchmarks/benchmark/tools/locust-load-inference/README.md:12]: ty for pointing this out, fixed!

File: benchmarks/benchmark/tools/locust-load-inference/README.md (hunk 2 above)
Comment (annapendleton) [reply benchmarks/benchmark/tools/locust-load-inference/README.md]: I updated all of them to ../../infra instead :)

**Review by andrewsykim — APPROVED** [review]

**Review by achandrasekar — APPROVED** [review]

**Pull request merged**